src/Schema.php

```diff
@@ -24,6 +24,10 @@ public function render(array $document): array
             if (isset($property['example'])) {
                 $properties[$property['name']]['example'] = $property['example'];
             }
+
```

Comment:
check out this:

```php
    public function render(array $document): array
    {
        $properties = [];
        $required = [];

        foreach ($document['properties'] as $property) {
            $properties[$property['name']]['type'] = PropertyTypeTranslator::translate($property['type']);

            if (isset($property['description'])) {
                $properties[$property['name']]['description'] = $property['description'];
            }

            if (isset($property['format'])) {
                $properties[$property['name']]['format'] = $property['format'];
            }

            if (isset($property['example'])) {
                $properties[$property['name']]['example'] = $property['example'];
            }

            if (isset($property['required']) && $property['required']) {
                $required = $property['name'];
            }
        }

        $message[$document['name']] = [
            'payload' => [
                'type' => 'object',
                'properties' => $properties,
                'required' => $required,
            ],
        ];

        return $message;
    }
```

Reply:
I think this should fix it to work as expected.